University-advisor-web/Controllers/SettingsController.cs

```diff
@@ -122,5 +125,14 @@ public IActionResult ChangeStatus(UserModel model)
             _errorHandler.ShowError(this, Messages.statusChangeSuccessfull, "Success");
             return RedirectToAction("Index", model);
         }
+
+        [HttpPost]
+        public IActionResult UploadFile(UserModel model)
+        {
+            var rec = new CardRecognition(_logger);
+            var result = rec.StartStudentCardValidation(model.File);
+            _errorHandler.ShowError(this, result.Information, "Information");
+            return RedirectToAction("Index", model);
+        }
```

Comment:
What will be shown to user if the validation failed? Shouldn't there be any error handling?

Reply:
For some reason file was not saved. I will fix that immediately.